src/main/java/com/impactupgrade/nucleus/service/segment/HubSpotCrmService.java

```diff
@@ -383,6 +399,24 @@ public void addContactToList(CrmContact crmContact, String listId) throws Except
     log.info("added HubSpot contact {} to list {}", crmContact.id, listId);
   }
 
+  @Override
```

Comment:
Essentially C/P from the addContactToList() prior to it. Everything look right here?